In [17]:
import pandas as pd


In [18]:
amba = pd.read_csv('./data/AMBA.csv')

In [19]:
print(amba.shape)

(2853, 7)


In [22]:
amba.head()

Date  Open  High   Low  Close  Adj Close  Volume
0  2012-10-11  6.06  6.44  6.00   6.08       6.08  416700
1  2012-10-12  6.05  6.30  6.01   6.23       6.23  155200
2  2012-10-15  6.35  6.40  6.05   6.15       6.15  156700
3  2012-10-16  6.15  6.30  5.94   6.01       6.01  173000
4  2012-10-17  6.00  6.08  5.99   6.00       6.00  113600

In [21]:
amba.describe()

Open         High          Low        Close    Adj Close  \
count  2853.000000  2853.000000  2853.000000  2853.000000  2853.000000   
mean     59.399888    60.739012    58.063505    59.403645    59.403645   
std      33.523402    34.412228    32.559028    33.471636    33.471636   
min       6.000000     6.080000     5.550000     6.000000     6.000000   
25%      40.040001    40.990002    39.150002    40.090000    40.090000   
50%      53.720001    54.740002    52.680000    53.680000    53.680000   
75%      70.199997    72.059998    68.570000    70.190002    70.190002   
max     216.660004   227.589996   209.050003   216.839996   216.839996   

             Volume  
count  2.853000e+03  
mean   1.004999e+06  
std    1.204699e+06  
min    1.890000e+04  
25%    4.201000e+05  
50%    6.554000e+05  
75%    1.116200e+06  
max    2.507260e+07

The min open/close price of AMBA is 6.00000
The max open/close price of AMBA is 216.660004 and 216.839996 respectively

In [23]:
amba.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2853 entries, 0 to 2852
Data columns (total 7 columns):
 #   Column     Non-Null Count  Dtype  
---  ------     --------------  -----  
 0   Date       2853 non-null   object 
 1   Open       2853 non-null   float64
 2   High       2853 non-null   float64
 3   Low        2853 non-null   float64
 4   Close      2853 non-null   float64
 5   Adj Close  2853 non-null   float64
 6   Volume     2853 non-null   int64  
dtypes: float64(5), int64(1), object(1)
memory usage: 156.2+ KB
